### PR TITLE
VER-81473 update package names and all repository references to updated names

### PR DESCRIPTION
**Pull request by DMickens**
This PR looks a bit crazy because of all the files changed. Here is a summary of what changed:

1. pg-connection-string, pg-cursor, pg-pool, pg-protocol, pg-query-stream, and pg became v-connectionString, v-cursor, VPool, v-protocol, v-queryStream, and vertica-nodejs respectively. 
2. All references to those name changes in tests were updated. Variable names related to these instances were updated as well. Ex. var pg = require('pg')  ----->  var vertica = require('vertica-nodejs')
3. Updated paths in Travis continuous integration files to reflect name changes
4. Updated package.json files to reflect name changes, repository changes, author, etc.

### CODE REVIEW

File: packages/VConnectionString/package.json [hunk 1]
```diff
@@ -11,21 +11,21 @@
...
+    "directory": "packages/VConnectionString"
   },
   "keywords": [
     "pg",
     "connection",
     "string",
     "parse"
   ],
-  "author": "Blaine Bublitz <blaine@iceddev.com> (http://iceddev.com/)",
+  "author": "Vertica",
```
Comment (alexey-temnikov) [inline packages/VConnectionString/package.json:23]: Shall we create a NOTICE file to attribute the original author?

File: .eslintrc [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "plugins": ["prettier"],
   "parser": "@typescript-eslint/parser",
   "extends": ["plugin:prettier/recommended", "prettier/@typescript-eslint"],
-  "ignorePatterns": ["node_modules", "coverage", "packages/pg-protocol/dist/**/*", "packages/pg-query-stream/dist/**/*"],
+  "ignorePatterns": ["node_modules", "coverage", "packages/VProtocol/dist/**/*", "packages/VQueryStream/dist/**/*"],
```
Comment (alexey-temnikov) [inline .eslintrc:5]: It looks we use both -`kebab-case` and `camelCase`.  Shall we unify the naming convention for node modules? `kebab-case` is the most common for nodejs module, however, it is not a requirement.

File: packages/VConnectionString/package.json (hunk 1 above)
Comment (DMickens) [reply packages/VConnectionString/package.json:23]: We certainly could, but I believe we are not required to with MIT license.

File: .eslintrc (hunk 2 above)
Comment (DMickens) [reply .eslintrc:5]: This change is fine with me. If going with a uniform convention I would prefer kebab-case. Probably should have just gone with this from the start. v-protocol, v-query-stream, etc are all available. I will make that update

Comment (williammatherjones) [reply .eslintrc:5]: I agree with the goal of using kebab-case

**Review by jonathanl-bq — APPROVED** [review]
If the other comments are addressed, then it seems fine to me. I didn't look at all 226 files though :P. I'm also assuming that you've successfully built and run the tests after the rename to make sure nothing has broken.

**Review by williammatherjones — CHANGES_REQUESTED** [review]
Change camelCase to kebab-case as Alexey suggested.

**Pull request merged**